recipe/bld.bat

```diff
@@ -15,3 +15,7 @@ if errorlevel 1 exit 1
 REM Install step
 nmake install
 if errorlevel 1 exit 1
+
+REM Rename static libraries to be consistent with conda-forge's jpeg recipe
+move %LIBRARY_PREFIX%\lib\jpeg-static.lib %LIBRARY_PREFIX%\lib\libjpeg.lib
+move %LIBRARY_PREFIX%\lib\turbojpeg-static.lib %LIBRARY_PREFIX%\lib\libturbojpeg.lib

```

Comment:
@valgur I am not sure about this. Do we really want them to clash?

Pinging @gillins here for another opinion.